app/repositories/EmploymentUserDataRepository.scala

```diff
@@ -44,9 +47,16 @@ class EmploymentUserDataRepositoryImpl @Inject()(mongo: MongoComponent, appConfi
   indexes        = EmploymentUserDataIndexes.indexes(appConfig),
   //TODO REMOVE REPLACE INDEXES WHEN DEPLOYED FULLY TO PRODUCTION
   replaceIndexes = true
-) with Repository with EmploymentUserDataRepository {
+) with Repository with EmploymentUserDataRepository with Logging {
 
-  def create[T](userData: EmploymentUserData)(implicit user: User[T]): Future[Boolean] = collection.insertOne(userData).toFutureOption().map(_.isDefined)
+  def create[T](userData: EmploymentUserData)(implicit user: User[T]): Future[Boolean] = {
+    collection.insertOne(userData).toFutureOption().map(_.isDefined).recover{
+      case e: Exception =>
+        pagerDutyLog(FAILED_TO_CREATE_EMPLOYMENT_DATA, s"[EmploymentUserDataRepositoryImpl][create] Failed to create employment user data. " +
+          s"Error:${e.getMessage} SessionId: ${user.sessionId}")
```

Comment:
do you need a fullstop between the Error:${e.getMessage} and SessionId for better readability?

Reply:
Updated